### DISCUSSION TITLE
ERR_MODULE_NOT_FOUND

### DISCUSSION THREAD

**Discussion by borkdude** [johnlindquist/kit#735] [Errors] [answered]
I tried to load [nbb](https://github.com/borkdude/nbb) in kit:

``` javascript
import "@johnlindquist/kit"
let nbb = await npm("nbb")

console.log(nbb);
```

and got:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/borkdude/.kenv/node_modules/nbb/index.js' imported from /Users/borkdude/.kit/api/npm.js
Error: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/borkdude/.kenv/node_modules/nbb/index.js' imported from /Users/borkdude/.kit/api/npm.js
    at kenvImport (file:///Users/borkdude/.kit/api/npm.js:42:15)
    at async file:///Users/borkdude/.kit/api/npm.js:58:12
    at async file:///Users/borkdude/.kenv/scripts/nbb-test.js

**Comment by johnlindquist** [comment, marked as answer]
Thanks for letting me know! Looks like a bug in the `npm` helper looking for the `.js` extension instead of the `.mjs` extension. Should be a simple fix for the next release.

For now, since the install probably succeeded, you can swap over to using `import`:

> `npm` installs if missing, then imports. If the package already exists, `npm` and `import` are essentially the same:

```js
import "@johnlindquist/kit"
let nbb = await import("nbb")
```

**Reply by borkdude** [reply]
Thanks.
